### PR TITLE
Non-interactive login command.

### PR DESCRIPTION
**Pull request by slashrsm**
Works only with 2FA disabled.

### CODE REVIEW

File: src/Command/Auth/LoginNonInteractiveCommand.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    protected function configure()
+    {
+        $this
+            ->setName('nilogin')
+            ->setDescription('Non-interactive log in to ' . self::$config->get('service.name'));
+        $help = 'Use this command to log in to your ' . self::$config->get('service.name') . ' account in a non-interactive way.'
+            . "\n\nYou can create an account at:\n    <info>" . self::$config->get('service.accounts_url') . '</info>'
+            . "\n\nIf you have an account, but you do not already have a password, you can set one here:\n    <info>" . self::$config->get('service.accounts_url') . '/user/password</info>';
+        $this->setHelp($help);
```
Comment (ralt) [inline src/Command/Auth/LoginNonInteractiveCommand.php]: Maybe add that TFA must be disabled?

Comment (slashrsm) [inline src/Command/Auth/LoginNonInteractiveCommand.php]: Done.

**Comment by pjcdawkins** [comment]
This would mean entering a password on the command-line, which would remain in shell history, etc., in plain text. Maybe that's fine on an otherwise secure environment, but that's not something we can assume.

There is "API token" support for this reason (see https://github.com/platformsh/platformsh-cli#customization).

**Comment by slashrsm** [comment]
Thanks! I guess this makes it PR useless.

**Pull request closed without merging**